### DISCUSSION TITLE
[Chapter 08] ORM

### DISCUSSION THREAD

**Discussion by ahngilwoong** [cowaine/spring-boot-for-msa#86] [Chapter 08. Data Persistence] [answered]
1. 책 내용에서 제가 처음 접한 단어인 OLTP와 OLAP라는 단어가 나옵니다.
JPA의 즉각적인 트랜잭션으로 OLTP에 대해서는 대충 이해가 되었습니다. 그런데
OLAP는 방대한 양의 비즈니스 데이터를 분석에 용이하도록 구성하는 것을 지원하는 정보 시스템의 한 계열이라고 하는데
책의 내용으로는 Mybatis 및 SQL Mapper가 OLAP다? 아니면 OLAP에 적합하다? 라고 이해하는게 맞는 걸까요?
그리고 OLAP는 읽기의 성능이 좋은 대량의 데이터를 조회할때 쓰면 좋다? 정도로 이해하면 맞을까요?
OLAP에 대해 개념이 정립된 분이 있다면 알려주시면 감사하겠습니다.
[OLTP ? OLAP?](https://co-no.tistory.com/65)

**Comment by corock** [comment, marked as answer]
ORM 같은 매핑 기술이 쿼리 조회 전에 한 계층으로 포함되는 것보다,
빅데이터 분석에는 대용량의 데이터 처리 및 조회가 더 중요하기 때문에 책에서 각 용도에 맞게 사용하자고 소개하는 것 같습니다.

OLAP 에 대해 아주 잘 알진 못하지만, 예를 들어보면 규모가 있는 기업은 보통 빅데이터를 다루는 팀이 별도로 있습니다.
빅데이터 팀에서 다루는 업무 중 하나로 Reporting Service 를 제공할 수 있을 것 같은데요.
이건 보통 고객/판매/매출 등 다양한 관점으로 실무에서 필요한 데이터를 각종 리포트로 제공하는 서비스를 말합니다.
이때, 사용하는 솔루션은 여러 가지가 있으나 제가 알고 있는 솔루션으로는 태블로(Tableau)라는 것이 있습니다.

2022년에 Tableau DataFest Seoul/Korea 2022 세미나가 있었는데, 그때 본 기조연설이 있어 관련 [링크](https://www.tableau.com/ko-kr/learn/on-demand/DataFest-Seoul-2022-08-25) 같이 첨부합니다.

**Comment by Irisation23** [comment]
### 질의 응답 이외의 CheckPoint

1. [OLTP](https://github.com/cowaine/spring-boot-for-msa/discussions/83) 과 같이 연계해서 정리해 볼 것
